src/helpers/promise.js

```diff
@@ -1,4 +1,5 @@
 /* eslint no-proto:0 */
+/* eslint no-use-before-define:0 */
```

Comment:
What did you prefer, as global into the ```.eslintrc``` or in the comments ? Only one file has this problem.